### ISSUE TITLE
Full Calendar css broken

### ISSUE THREAD

**Issue by mlengse** [BayBreezy/ui-thing#21]
![image](https://github.com/user-attachments/assets/866cf8fb-80d0-454e-b536-08ea34efb341)

Hi Sir Behon,

in the [full calendar documentation](https://ui-thing.behonbaker.com/goodies/full-calendar#css), after looking at the code, then going back to the preview, it looks like the CSS arrangement is broken. The same goes if the code is run in my setup. Is there a way to fix it? Thanks.

**Comment by BayBreezy** [comment]
Hey @mlengse , I actually saw this but I ignored it lolololol
The issue is whit how the tabs work. The info inside each tab content is unMounted when you switch between the different tabs and that is causing the problem. 
To fix it, I would have to tell the showcase component that it should not unmount the content but that would lead to performance issues.
So I had to choose between performance OR the user refreshing after switching between tabs.
At the end of the day, its better if you just refresh the page. I know, the UX is horrible but I think its better than slowing down the site
